### DISCUSSION TITLE
Translation memory search doesn't work

### DISCUSSION THREAD

**Discussion by ELatestark** [WeblateOrg/weblate#12530] [Q&A]
Hey!
I do have translation memory (we recently migrated from Zanata):
![368672181-cd9b7ee0-574c-44c2-a026-12d0cbb1eff4](https://github.com/user-attachments/assets/e386ca98-02b9-402d-b87c-764ff7cefa02)
But when I try to translate something translation memory show no matches:
![Снимок экрана от 2024-09-19 00-09-25](https://github.com/user-attachments/assets/6348f4ae-2e78-40e6-a52c-145360021f76)
I'm 100% sure that there are matches in those 13000 entries. Is the threshold set too high? Why it's not showing particular words and phrases? If its threshold, can I set it by myself?
If not - whats the workaround?
I've installed add-on for automatic translations into the project and set threshold for 15%:
![Снимок экрана от 2024-09-19 13-29-24](https://github.com/user-attachments/assets/820bce7a-7af2-42b1-90b8-8f5182df6356)
, but the result is the same - no suggestions from translation memory.

**Comment by nijel** [comment]
Is the translation memory screenshot for the same project as you are translating (it has project name wiped out)?

See https://docs.weblate.org/en/latest/admin/memory.html for info on translation memory scopes.

**Reply by ELatestark** [reply]
Yes, screenshot of a translation memory was taken from project page, then Manage button -> Translation memory.
Dunno what should I look for.

**Reply by nijel** [reply]
Few things to look at:

* Are there really some matches? Try downloading the memory and look there.
* Does rebuild help?
* What database (including version) do you use?

**Reply by ELatestark** [reply]
I've downloaded tmx file locally and opened it with Notepad.
First word from screenshot where I'm trying to get some suggests is "Указывает" and it has 80 entries in tmx.
![Снимок экрана от 2024-09-27 23-38-08](https://github.com/user-attachments/assets/1648c9b3-7a86-4576-9791-70f93b41e486)
After rebuilding Translation Memory nothing has changed.
I'm using weblate within docker compose, which runs postgres 16.4 in alpine-based image.
